onboarding-enabler-spring/src/main/java/com/ca/apiml/enable/config/ApiMediationServiceConfigBean.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.apiml.enable.config;
+
+import com.ca.mfaas.eurekaservice.client.config.ApiMediationServiceConfig;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("apiml.service")
+public class ApiMediationServiceConfigBean extends ApiMediationServiceConfig {
```

Comment:
Can you do this without subclassing and create the bean from the original class instead? You can use the @ConfigurationProperties on the @Bean with the same effect. This applies for the other bean as well

Reply:
@Bean can't be applied to the original class, because it is part of PJE module, which does not have dependency on Spring at all. We could use an other @Configuration class to create @Bean and apply the @ConfiturationProperties to that bean. This could be closer to my thinking. The current approach was chosen by Petr Plavjanik in the APIML SDK and Jirka reused it here. Since it is valid approach and works I didn't change it.

Reply:
yes, you are correct and i did not describe it well enough to communicate my intent. Subclassing works, but creates additional classes called ****Bean, which is nothing than the original class. I see no value in creating such class and obfuscating the original one. But it is not broken and it's working.

Reply:
I'll test the other approach and if it does not bring any unexpected negative consequences will commit it.

Reply:
Done, additional classes removed and beans created by shared existing @Configuration class for the module.

Reply:
Did it. Works fine.